js/script.js

```diff
@@ -6,16 +6,44 @@ let tasks = [];
 const addButtonHandler = () => {
   const taskTitle = $taskInput.value.trim();
   if (taskTitle) {
-    tasks.unshift(taskTitle);
-    const $tasksList = document.createElement("li");
-    $tasksList.textContent = taskTitle;
-    $taskList.prepend($tasksList);
-    $taskInput.value = "";
+    createTask(taskTitle);
     showToastMessage("Task added successfully!");
     $addButton.disabled = true;
   }
 };
 
+const deleteHandler = (taskId) => {
+  tasks = tasks.filter((task) => task.id !== taskId);
+  renderTasks();
+};
+
+const createTask = (taskTitle) => {
+  const task = {
+    id: new Date().getTime(),
+    title: taskTitle,
+  };
+  tasks.unshift(task);
+  renderTasks();
+
+  $taskInput.value = "";
+};
+
+const renderTasks = () => {
+  $taskList.innerHTML = "";
+
+  tasks.forEach((task) => {
+    const tasksList = document.createElement("li");
+    tasksList.textContent = task.title;
+
+    const deleteButton = document.createElement("button");
```

Comment:
We can modularize these elements which would be created.

Reply:
Resolved.